tactics/docker.py

```diff
@@ -0,0 +1,22 @@
+from charmtools.build.tactics import WheelhouseTactic
+
+
+class DockerWheelhouseTactic(WheelhouseTactic):
+    def __call__(self, venv=None):
+        # this was due to some weird scoping issue during execution.
+        # investigate more later
+        from charmtools import utils
+        create_venv = venv is None
+        venv = venv or path(tempfile.mkdtemp())
```

Comment:
Actually @marcoceppi we found out that the utils.load_class() method was loading this library with it's global namespace/libraries and we were able to use path, tempfile and other libraries without importing them.